### ISSUE TITLE
go get github.com/docker/swarm broken

### ISSUE THREAD

**Issue by gongsu832** [docker/swarm#2111]
It was working yesterday but today:

```
root@zhlpeer:/opt# export GOPATH=/opt/golang
root@zhlpeer:/opt# go get github.com/docker/swarm
# github.com/docker/swarm/api/nopclient
golang/src/github.com/docker/swarm/api/nopclient/nop.go:226: undefined: client.RequestPrivilegeFunc
golang/src/github.com/docker/swarm/api/nopclient/nop.go:231: undefined: client.RequestPrivilegeFunc
golang/src/github.com/docker/swarm/api/nopclient/nop.go:241: undefined: client.RequestPrivilegeFunc
```

**Comment by MHBauer** [comment]
I think if `go get` worked, it was by chance that all the dependencies were at compatible. The official way to build is with `godep` and documented in [contributing](https://github.com/docker/swarm/blob/master/CONTRIBUTING.md).

**Comment by gongsu832** [comment]
By chance or not, go get github.com/docker/swarm certainly is a much simpler way to build than the "official" way.

**Comment by vieux** [comment]
@gongsu832 it's very common to have this issue when the project's dependencies evolve so quickly.

This particular issue is going to be solved when we are going to switch to go 1.6 with the new vendoring

**Comment by gongsu832** [comment]
@vieux good to know. Thanks!

**Comment by nishanttotla** [comment]
This is due to https://github.com/docker/engine-api/pull/162 being merged today. This should be fixed the next time we update Godeps for `engine-api`, preferably after #1983.

**Comment by vieux** [comment]
@gongsu832 this will be fixed by #2124
